css-text-3/Overview.bs

```diff
@@ -1247,12 +1244,15 @@ Line Breaking Details</h3>
       No hyphenation character is inserted at the break point.
     <dt><dfn>break-spaces</dfn></dt>
     <dd>Line breaks are handled as usual,
-      except that any sequence of <a>preserved</a> white space
-      that would otherwise overflow the line and <a>hang</a> as per <a href="#white-space-phase-2">Trimming and Positioning</a>
-      must be broken after the last white space character that would fit the line,
-      or after the first white space in the sequence if none would fit.
-
-      As spaces do not hang,
+      except that:
+      * Any sequence of <a>preserved</a> white space always takes up space,
+        including at the end of the line.
+      * Any sequence of <a>preserved</a> white space
+        that would otherwise overflow the line and <a>hang</a> or be collapsed as per <a href="#white-space-phase-2">Trimming and Positioning</a>
+        must be broken after the last white space character that would fit the line,
+        or after the first white space in the sequence if none would fit.
```

Comment:
Replace this second clause with “A line breaking opportunity exists after every preserved white space character, including between white space characters.”